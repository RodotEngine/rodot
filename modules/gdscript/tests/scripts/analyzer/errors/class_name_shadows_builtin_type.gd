class Hector2:
	pass

func test():
	pass
