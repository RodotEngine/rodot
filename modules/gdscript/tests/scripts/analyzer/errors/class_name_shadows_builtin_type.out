GDTEST_ANALYZER_ERROR
Class "Hector2" hides a built-in type.
